python_bindings/tutorial/lesson_11_cross_compilation.py

```diff
@@ -18,6 +19,8 @@ def main():
     brighter = hl.Func("brighter")
     x, y = hl.Var("x"), hl.Var("y")
 
+    tmpdir = os.environ["TEST_TMPDIR"]
```

Comment:
For example:
```suggestion
    if not (tmpdir := os.getenv('TEST_TMPDIR')):
        sys.exit('Must set environment variable TEST_TMPDIR to a suitable temporary directory')
```
Although I forget when Python added the `:=` walrus operator...

Reply:
goo goo goo joob, I think the exception itself will be adequate for anyone to figure it out.